src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.querySelectorAll('tr')];
+
+for (const row of rows) {
+  const newFootTh = document.createElement('th');
+  const tfoot = document.querySelector('tfoot');
```

Comment:
The `tfoot` element is being queried inside the loop for each row. It would be more efficient to move this query outside the loop to avoid unnecessary DOM queries.